docs/sources/shared/datasources/tempo-search-traceql.md

```diff
@@ -161,9 +161,12 @@ Queries can take a little while to return results. The results appear in a table
 The Tempo data source supports streaming responses to TraceQL queries so you can see partial query results as they come in without waiting for the whole query to finish.
 
 {{% admonition type="note" %}}
-To use this experimental feature, enable the `traceQLStreaming` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.
+To use this experimental feature, enable the `traceQLStreaming` feature toggle. When active, all configured Tempo data sources will attempt to use streaming.
+
+In Grafana Cloud, you can enable this capability in the **Streaming** section of the Tempo data source settings.
 {{% /admonition %}}
 
-Streaming is available for both the **Search** and **TraceQL** query types, and you'll get immediate visibility of incoming traces on the results table.
+Streaming is available for both the **Search** and **TraceQL** query types.
```

Comment:
```suggestion
Results will be streamed when using both the **Search** and **TraceQL** query interfaces in Explore.
```

Reply:
I don't really think of them as different "types" so much as different ways to run queries. One is more like a WIZYWIG way to build queries / graphical query builder, and the other is more like the command-line where you type the query into a blank text box.

Reply:
The UI calls them query types. 
![image](https://github.com/user-attachments/assets/630c2e68-68c4-4626-bb29-8bb73836f806)

Reply:
Ah good point. Resolving then.